flink-ml-lib/src/test/java/org/apache/flink/ml/clustering/OnlineKMeansTest.java

```diff
@@ -407,13 +415,15 @@ public void testSaveAndReload() throws Exception {
 
         transformAndOutputData(loadedOnlineModel);
 
-        miniCluster.submitJob(env.getStreamGraph().getJobGraph());
-        waitInitModelDataSetup();
+        final JobSubmissionResult jobSubmissionResult =
+                miniCluster.submitJob(env.getStreamGraph().getJobGraph()).get();
+        final JobID jobID = jobSubmissionResult.getJobID();
```

Comment:
nit: I noticed that some tests uses 
```java
final JobSubmissionResult jobSubmissionResult =
        miniCluster.submitJob(env.getStreamGraph().getJobGraph()).get();
final JobID jobID = jobSubmissionResult.getJobID();
```
while others use
```java
final JobGraph jobGraph = env.getStreamGraph().getJobGraph();
miniCluster.submitJob(jobGraph);
final JobID jobID = jobGraph.getJobID();
```
And I think it might be better to unify the practice across all implementations.

Reply:
Generally, I'd like to avoid using `Future.get()` without a timeout. Is it ok to assume 1 second? 
I could chain the test as well, but this may make the code way harder to follow.

Reply:
updated